src/lib/message/L1ToL2MessageGasEstimator.ts

```diff
@@ -11,6 +11,7 @@ import { constants } from 'ethers'
 import { utils } from 'ethers'
 
 const DEFAULT_SUBMISSION_PRICE_PERCENT_INCREASE = BigNumber.from(340)
+const DEFAULT_MAX_GAS_PERCENT_INCREASE = BigNumber.from(50)
```

Comment:
Lets a comment as to why this is necessary, so that people know when/if they can remove it.

Reply:
```suggestion
// Temporary workaround for incorrect gas estimation from NodeInterface when there is gas refund
const DEFAULT_MAX_GAS_PERCENT_INCREASE = BigNumber.from(50)
```